src/main/java/frc/robot/vision/limelights/GyroAngleValues.java

```diff
@@ -0,0 +1,14 @@
+package frc.robot.vision.limelights;
+
+
+/**
+ * a record represents the gyros angle passed into the limelight's NT tables, to the robot_orientation_set entry. its values found
+ * in limelightlib sourcecode.
+ */
+public record GyroAngleValues(double yaw, double yawRate, double pitch, double pitchRate, double roll, double rollRate) {
+
+	public double[] getAsArray() {
```

Comment:
If we have a record for this, why do we need to use an array?

Reply:
the Network tables robotOrientationEntry gets a double array, the record is in use only to be more readable and so it's clear what each parameter represents